packages/web/components/cards/pool-card.tsx

```diff
@@ -0,0 +1,197 @@
+        <div
+          className={
+            "absolute z-10 w-[4.125rem] h-[4.125rem] rounded-full border-[1px] bg-card border-enabledGold flex items-center justify-center"
+          }
+        >
+          {pool.poolAssets[0].amount.currency.coinImageUrl ? (
+            <Image
+              src={pool.poolAssets[0].amount.currency.coinImageUrl}
+              alt={pool.poolAssets[0].amount.currency.coinDenom}
+              width={54}
+              height={54}
+            />
+          ) : (
+            <div
+              className={classNames(
+                "w-[54px] h-[54px] bg-blue rounded-full flex items-center justify-center",
+                poolCardIconBackgroundColorsToTailwindBgImage[
+                  PoolCardIconBackgroundColors[
+                    deterministicInteger % PoolCardIconBackgroundColors.length
+                  ]
+                ]
+              )}
+            >
+              <Image
+                src="/icons/OSMO.svg"
+                alt="no token icon"
+                width={40}
+                height={40}
+              />
+            </div>
+          )}
+        </div>
+        <div
+          className={
+            "ml-10 mr-6 w-[4.125rem] h-[4.125rem] rounded-full border-[1px] border-enabledGold shrink-0 flex items-center justify-center"
+          }
+        >
+          {pool.poolAssets.length >= 3 ? (
+            <div className="body1 text-white-mid ml-2.5">{`+${
+              pool.poolAssets.length - 1
+            }`}</div>
+          ) : pool.poolAssets[1].amount.currency.coinImageUrl ? (
+            <Image
+              src={pool.poolAssets[1].amount.currency.coinImageUrl}
+              alt={pool.poolAssets[1].amount.currency.coinDenom}
+              width={52}
+              height={52}
+            />
+          ) : (
+            <div
+              className={classNames(
+                "w-[54px] h-[54px] bg-blue rounded-full flex items-center justify-center",
+                poolCardIconBackgroundColorsToTailwindBgImage[
+                  PoolCardIconBackgroundColors[
+                    deterministicInteger % PoolCardIconBackgroundColors.length
+                  ]
+                ]
+              )}
+            >
+              <Image
+                src="/icons/OSMO.svg"
+                alt="no token icon"
+                width={40}
+                height={40}
+              />
+            </div>
+          )}
+        </div>
+        <div className="flex flex-col">
+          <h5>
+            {pool.poolAssets.length >= 3
+              ? `${pool.poolAssets.length} Token Pool`
+              : pool.poolAssets
+                  .map((asset) => asset.amount.currency.coinDenom)
+                  .join(" / ")}
+          </h5>
+          <div className="subtitle2 text-white-mid">{`Pool #${pool.id}`}</div>
+        </div>
+      </div>
+      <div className="mt-5 mb-3 w-full bg-secondary-200 h-[1px]" />
+      <div className="flex flex-wrap gap-x-8">
```

Comment:
For all these metrics below the green line in the card view -  @victortokaito - should we space evenly? Or justify left with some padding? I'm voting to space evenly. Tailwind would take `place-content-between` [docs](https://tailwindcss.com/docs/place-content#space-between)

Reply:
The discussion seems to be getting longer, so I think we can resolve this on implementing pools page.